### DISCUSSION TITLE
功能增加建议

### DISCUSSION THREAD

**Discussion by GH6324** [yangjian102621/chatgpt-plus#90] [规划中的任务]
1.开启邮箱注册

2.打开首页推送公告（可以用Markdown语法来编辑）类似下图这种
![image](https://github.com/yangjian102621/chatgpt-plus/assets/47879885/5bfab484-43ac-48c5-913f-95b390b48497)

3.管理后台可以看到用户的对话信息【预设角色+提问问题+回复内容+模型+消耗的次数+时间+状态（是否请求成功）+操作（是否删除）】类似下图这种
![image](https://github.com/yangjian102621/chatgpt-plus/assets/47879885/c1d43ade-5b39-4ed3-b4cf-29a04221d24b)

4.key能添加同一个，而不是聊天和绘画只能使用不同的key

5.对每个OpenAi的key都能单独使用代理url，而不是后台只能指定1个url，然后key可以添加一个启用的开关
假如第三方中转的key出现问题，灾备切换会好很多
例如  key1：sk-123用的中转url是123.com    默认启用
         key2：sk-456用的中转url是456.com    默认关闭
         当123.com出现故障，马上在后台把使用123.com的key1关闭，然后把key2启用
![image](https://github.com/yangjian102621/chatgpt-plus/assets/47879885/3a9377fc-14a1-4a05-a749-d531d5d789d5)

**Comment by yangjian102621** [comment]
感谢反馈，除了公告，其他的功能的都可以考虑增加。公告不是必须的功能，我需要慎重考虑是否增加该功能。

**Comment by GH6324** [comment]
> 感谢反馈，除了公告，其他的功能的都可以考虑增加。公告不是必须的功能，我需要慎重考虑是否增加该功能。

好的，再次感谢您的无私奉献，请收下我的膝盖
